migrations/913-abuse-report-viewers.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO groups (name, rules, created, modified, notes) VALUES
+('App Abuse Viewers', 'Apps:ReadAbuse', NOW(), NOW(), 'Users who can mark read app abuse reports'),
+('Website Abuse Viewers', 'Websites:ReadAbuse', NOW(), NOW(), 'Users who can mark read website abuse reports');
```

Comment:
Reading this as "Users who can <verb> <verb> website abuse reports" was confusing until I figured out what you meant.